src/components/Favorites/index.js

```diff
@@ -0,0 +1,132 @@
+import React, { Component, Fragment } from 'react'
+import Nav from '../Nav'
+import SearchBar from '../SearchBar'
+import Gifs from '../Gifs'
+
+import axios from 'axios'
+
+class Favorites extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      trendingGifs: [],
+      query: '',
+      search: '',
+      loading: false,
+      title: '',
+      favoriteGifs: []
+    }
+  }
+
+  componentDidMount() {
```

Comment:
you should do this with sagas not with state. Remember that you are required to create almost stateless components.